dms_version/models/storage.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2017-2020 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import bsdiff4
+except ImportError:
+    bsdiff4 = False
```

Comment:
```suggestion
import bsdiff4
```
If you define `external_dependencies` in manifest about it not necessary make try except (because except is impossible and because it's impossible to codecov check too)